.travis.yml

```diff
@@ -18,7 +18,9 @@ jobs:
         # run single-threaded because some tests change environment variables, which is not thread-safe
         cargo test --package notion-core --package notion-fail --package node-archive --package notion-fail-derive --package progress-read -- --test-threads=1 &&
         # the acceptance tests, using network mocks
-        cargo test --features mock-network
+        cargo test --features mock-network &&
```

Comment:
Heads up, this'll be a tiny merge conflict since I merged #191.

Reply:
Yup, fixed that, and some other merge conflicts from the latest PR merges.